### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by lfcj**
Include directory to run command.
Also included `./sbt ...` as an option to be run, since some linux computers will only recognize it that way.

### CODE REVIEW

File: finagle-example/README [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # How to run the examples:
 
-From the command line:
+From the command line in cloned directory `finangle`:
```
Comment (mosesn) [inline finagle-example/README]: git doesn't require that the cloned directory be named finagle, maybe we should say the root directory of the finagle repository?

**Comment by mosesn** [comment]
I think the assumption with using raw sbt was that you have sbt installed on your machine.  Instead of using ||, it's probably fine to just make the examples use ./sbt by default.

**Comment by mosesn** [comment]
Thanks for the contribution :]

File: finagle-example/README (hunk 1 above)
Comment (spockz) [inline finagle-example/README]: Also there is a typo in `finangle`, it should be `finagle`.

**Comment by dschobel** [comment]
@lfcj are you still interested in working on the open issues in this PR?

**Comment by dschobel** [comment]
closing this until we hear back.

**Pull request closed without merging**